battDB/models.py

```diff
@@ -439,6 +439,19 @@ class Experiment(cm.BaseModel):
         " If 'other', technique should be specified in the notes section.",
     )
 
+    external_link = models.URLField(
+        blank=True,
+        null=True,
+        help_text="Specific link to a reference for this experiment.",
+    )
+
+    summary = models.TextField(
+        null=True,
+        blank=True,
```

Comment:
Both of these should be false. From conversation with the researchers, this is like a mandatory description of the experiment. In fact, I would go as far as using the [MinLengthValidator](https://docs.djangoproject.com/en/4.1/ref/validators/#minlengthvalidator) to ensure they write at least 20 characters or something (so a few words).